pkg/ansible/run.go

```diff
@@ -89,7 +89,7 @@ func Run(flags *aoflags.AnsibleOperatorFlags) error {
 
 	var gvks []schema.GroupVersionKind
 	cMap := controllermap.NewControllerMap()
-	watches, err := watches.Load(flags.WatchesFile, flags.MaxWorkers)
+	watches, err := watches.Load(flags.WatchesFile, flags.MaxWorkers, flags.AnsibleVerbosity)
```

Comment:
:+1:  while it is more verbose, no pun intended, it is very explicit in what Load will need. Good job here.